core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -146,6 +146,11 @@ public class SchemaRegistryConfig extends RestConfig {
    */
   public static final String LEADER_ELECTION_DELAY = "leader.election.delay";
   public static final boolean DEFAULT_LEADER_ELECTION_DELAY = false;
+  /**
+   * <code>leader.election.sticky</code>*
+   */
+  public static final String LEADER_ELECTION_STICKY = "leader.election.sticky";
```

Comment:
Is the plan to enable this based on a LD flag in cc-spec-sr?

Reply:
yes